File: DataStructures/C++/binarytree.cpp
#include<bits/stdc++.h>
using namespace std;

class s{
    public:
    int data;
    s* left;
    s* right;

    s(int val){
        data = val;
        left = NULL;
        right = NULL;
    }
};

int main(){
    s* root = new s(1);

    root->left = new s(2);
    root->right = new s(3);

    root->left->left = new s(4);
    root->left->right = new s(10);
    return 0;
}

File: DataStructures/Java/binarytree.java
class Nodes{
    int key;
    Nodes left, right;

    public Nodes(int item){
        key = item;
        left = right = null;
    }
}

class binarytree{
    Nodes root;

    binarytree(int key){
        root = new Nodes(key);
    }
    binarytree(){
        root = null;
    }
    public static void main(String[] Args){
        binarytree tree = new binarytree();
        tree.root = new Nodes(3);
        tree.root.left = new Nodes(4);
        tree.root.right = new Nodes(5);
    }
};